src/citrine/informatics/reports.py

```diff
@@ -96,8 +96,8 @@ class ModelSummary(Serializable['ModelSummary']):
 
     name = properties.String('name')
     type_ = properties.String('type')
-    inputs = properties.List(properties.String(), 'inputs')
-    outputs = properties.List(properties.String(), 'outputs')
+    inputs = properties.List(properties.Union([properties.Object(Descriptor), properties.String()]), 'inputs')
```

Comment:
I'd like to make this just `inputs = properties.List(properties.Object(Descriptor), 'inputs')` instead of this union, but doing so breaks a bunch of tests and backwards compatibility.